pkg/backporter/backporter.go

```diff
@@ -311,40 +356,46 @@ func getClonesTemplateData(bugID int, client bugzilla.Client, allTargetVersions
 	if err != nil {
 		return nil, http.StatusNotFound, fmt.Errorf("Bug#%d not found: %w", bugID, err)
 	}
-
-	prs, err := client.GetExternalBugPRsOnBug(bug.ID)
-	if err != nil {
-		return nil, http.StatusInternalServerError, fmt.Errorf("Bug#%d - error occured while retreiving list of PRs: %w", bug.ID, err)
-	}
-
-	parent, err := client.GetRootForClone(bug)
-	if err != nil {
-		return nil, http.StatusInternalServerError, fmt.Errorf("unable to fetch root bug: %w", err)
-	}
-
 	clones, err := client.GetAllClones(bug)
 	if err != nil {
 		return nil, http.StatusInternalServerError, fmt.Errorf("unable to get clones: %w", err)
 	}
+	if len(clones) < 1 {
+		return nil, http.StatusInternalServerError, fmt.Errorf("clones list empty")
```

Comment:
Why is this an error condition? A bug doesn't need to have clones?

Reply:
The current logic for clones should return at least one bug (ie itself) in the clones list, since we have now moved to everything in the table view.